triagebot.toml

```diff
@@ -13,7 +13,26 @@ allow-unauthenticated = [
     "D-*",
     "E-*",
     "F-*",
-    "I-*",
+    "I-compilemem",
+    "I-compiletime",
+    "I-ICE",
+    "I-heavy",
+    "I-crash",
+    "I-unsound",
+    "I-slow",
+    "I-needs-decision",
+    "I-hang",
+    "I-prioritize",
+    "I-monomorphization",
+    "I-cycle",
+    "I-memleak",
+    "I-miscompile",
+    "I-lang-radar",
+    "I-lang-bikeshed",
```

Comment:
I think these 2 are "reserved" to T-lang members. Specifically @traviscross uses them